copy_with_style.js

```diff
@@ -69,21 +71,21 @@ class Copy_With_Style {
 	//  a list of two ints describing [threshold, frequency]
 	// 		where we defer to UI only if there are more than threshold elements
 	//		and only after batches of "frequency" elements are inlined.
-	//      a frequency of 0 request optimisation (i.e. number of elements per progress bar pixel)
+	//      a frequency of 0 request optimization (i.e. number of elements per progress bar pixel)
```

Comment:
Actually we use s (English spelling, z is 'murican) But I don't really care, but it's likely I see English versions all over